actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -716,6 +724,14 @@ def convert_options_to_data_attributes(options, html_options)
           end
         end
 
+        def url_target(name, options)
+          if name.respond_to?(:model_name) && options.empty?
+            url_for(name)
+          else
+            url_for(options)
```

Comment:
Funny story: an app that is just getting upgraded to Rails 7 actually relied on the old behavior of `link_to(active_record_model)` where `Model#to_s` was used for the link name and the URL was set to `url_for({})` which returns the current path. I didn't see any mention of this in the upgrade guide or release notes, but I found this by looking at the git history for `link_to`. 😆

Reply:
It is in the changelog. Not everything from the CHANGELOG makes to those notes. Only things that would impact most of people and that need explanation how to solve. But, I think we can include this one. Do you want to open a PR to add it to the release notes?

Reply:
@rafaelfranca I think the current judgment is sound, this probably doesn't impact many people as there wasn't really a good reason to intentionally do `link_to(active_record_model)` before and expect the current path instead of the model's path. Also Rails 7 is several years old now, I think if no one else encountered this first it's unlikely to come up again.

I really was just trying to share what I thought was a funny moment about working with weird legacy code, not trying to criticize any decision made by Rails. Thank you for your work!